error_reporting/unit_tests/test_util.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2016 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+
+
+class TestFlaskHelper(unittest.TestCase):
+
+    def test_flask_helper(self):
+        from google.cloud.error_reporting.util import build_flask_context
```

Comment:
Please define a `_call_fut` or `_call_function_under_test` method on this testcase class